app/views/partials/theme/simple/marketplace_setup/index.liquid

```diff
@@ -31,3 +31,36 @@
       <button class="mt-3 button">{{ 'app.stripe.save' | t }}</button>
     </form>
   </div>
+  </br>
+
+  <div class="mb-2 w-full md:w-full">
+    <h1 class="text-2xl">Chat</h1>
+  </div>
+
+  <div class="w-full p-6 mt-8 mb-2 bg-gray-200 rounded lg:w-4/6 md:w-3/4">
+    <form action="/marketplace_setup/constants/set" method="post">
+      {% include 'authenticity_token' %}
+      <input type="hidden" name="name" value="chat">
+      <input type="radio" class="mt-3" name="value" value="true" id="chatOn"{% if chat_enabled %}checked{% endif %}>
```

Comment:
`id="chatOn"{% if chat_enabled %}checked{% endif %}`

Missing space - if true, it will evaluate to `id="chatOn"checked` which should be `id="chatOn" checked`